---
bool isConnected
bool isProtectiveStopped
bool isEmergencyStopped
bool isSteady